ci/deploy/dockerhub-deploy.bash

```diff
@@ -29,7 +29,9 @@ make info-images
 
 # re-tag build
 DOCKER_IMAGE_TAG="$TAG_PREFIX-latest"
+BUILD_TARGET="fake" # this gets overwritten but is required when merging yaml files
```

Comment:
maybe "undefined" is a bit less scary?

Reply:
or something that makes it fail? like a tag with improper characters?

Reply:
sure, sounds better